website/blog/2022-09-06/index.md

```diff
@@ -49,13 +50,15 @@ The theme for this week is Azure Functions. We'll talk about ...
 
 
 ## Azure Functions and Serverless explained.
```

Comment:
Overall, let's assume the reader has zero exposure to serverless computing. We'll want to give them a brief overview of what & why it is. Then how & why Azure Functions solves that problem. 

A solid value add for Azure Functions to talk about up front is how you don't have to maintain objects controlled via triggers or bindings - all done by the runtime.

In particular for .NET devs - not only C#, but also Visual Studio integration. From templates to Connected Services.